### ISSUE TITLE
CNV state 4 VS expression data - don't match..

### ISSUE THREAD

**Issue by beemilmlo** [broadinstitute/infercnv#272]
Hello!

I've run infercnv and from the output files, there seemed to have been a total of about 15 CNVs inferred - ranging from states 2 ~ 4. From both my raw and normalized data (the raw counts matrix was used as input for infercnv), it seems the reference group of cells have an all around higher expression profile than the "tumor group cells" (they're not actually tumor cells, but just the group of cells being compared against the reference group). How is it that there is a CNV state 4 then? The average expression of the reference group cells about the CNV state 4 genes are higher than the average expression of the group of cells being observed as well.

An additional question is - CNV state 3 means neutral..so all the other regions of the genome other than the CNV 1, 2, 4, 5, 6 regions should be state 3, isn't that right? But the output predictions for my data show only CNV state 3 for a region on chromosome 6, and it's not like all the other regions are marked having CNVs either.

Best,
beemilmlo

**Comment by GeorgescuC** [comment]
Hi @beemilmlo ,

If the cells you provide as the reference have an all around higher expression, it likely is due to a difference in sequencing quality/depth, which gets normalized. What does the final infercnv.png figure look like?

A CNV with state 3 means it was initially predicted as a CNV at step 17, but later filtered out by the bayesian network at step 18.

Regards,
Christophe.

**Comment by beemilmlo** [comment]
Hi Christophe,

![image](https://user-images.githubusercontent.com/71161681/98208772-38da3980-1f81-11eb-8a13-8876d55a2379.png)
Here's what the image looks like.

I've also come across another interesting observation - I ran infercnv with the same input files 3 times, just with different variable names for the object and different output directory names, everything else was left the same. But I got three different results for the number of cnv genes; around 850 the first time, then 666 the second time, and around 1200 in the last. Might I be doing something wrong? I ran it a third time because the second was different than the first, expecting the third result to be the same with at least either one, but the third time showed a new set of genes.

Thanks a lot!
beemilmlo
